jina/docker/hubio.py

```diff
@@ -67,8 +67,10 @@ def _load_docker_client(self):
             # low-level client
             self._raw_client = APIClient(base_url='unix://var/run/docker.sock')
 
-    def new(self) -> None:
-        """Create a new executor using cookiecutter template."""
+    def new(self, no_input: bool = False) -> None:
+        """Create a new executor using cookiecutter template.
+            :param no_input: Argument to avoid prompting dialogue (just to be used for testing)
+        """
```

Comment:
reference: https://docs.jina.ai/chapters/docstring/docstring.html#multi-line-docstrings-example-of-a-function

```suggestion
        """
        Create a new executor using cookiecutter template.
        
        :param no_input: Argument to avoid prompting dialogue (just to be used for testing)
        """
```